amarna/command_line.py

```diff
@@ -149,9 +150,12 @@ def main() -> int:
 
     filename = args.filename
 
-    if not os.path.isabs(filename):
+    if filename is not None and not os.path.isabs(filename):
         filename = os.path.join(os.getcwd(), filename)
 
+    if not os.path.exists(filename):
+        raise Exception("The specified file doesn't exist")
```

Comment:
Instead of Exception, just print that message together with the filename and exit(-1)